recipes/centos_go/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM registry.centos.org/centos/centos
+
+RUN yum -y update && \
+    yum -y install sudo openssh-server g++ gcc glibc-devel make golang && \
+    yum clean all && \
+    sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config && \
+    sed -ri 's/#UsePAM no/UsePAM no/g' /etc/ssh/sshd_config && \
+    echo "%wheel ALL=(ALL) NOPASSWD: ALL" >> /etc/sudoers && \
+    useradd -u 1000 -G users,wheel -d /home/user --shell /bin/bash -m user && \
+    usermod -p "*" user && \
+    sed -i 's/requiretty/!requiretty/g' /etc/sudoers
+
+ENV GOPATH /go
+ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
```

Comment:
Done. Also moved `ENV` instruction to earlier in the file to reduce one layer due to `RUN` instruction.